src/Meta/genbroydenb.jl

```diff
@@ -0,0 +1,27 @@
+genbroydenb_meta = Dict(
+  :nvar => 100,
+  :variable_nvar => true,
+  :ncon => 0,
+  :variable_ncon => false,
+  :minimize => true,
+  :name => "genbroydenb",
+  :has_equalities_only => false,
+  :has_inequalities_only => false,
+  :has_bounds => false,
+  :has_fixed_variables => false,
+  :objtype => :nls,
```

Comment:
```suggestion
  :objtype => :nls,
```
please check that as asked for the others